src/util/migrate.ts

```diff
@@ -27,20 +29,21 @@ export default (async () => {
                 const migration = require(migrationPath as string);
                 return {
                     name,
-                    up: async () => migration.up(context),
-                    down: async () => migration.down(context)
+                    up: async () => migration.up(context, Sequelize),
+                    down: async () => migration.down(context, Sequelize)
                 };
             }
         },
         context: sequelize.getQueryInterface(),
         storage: new SequelizeStorage({sequelize}),
-        logger: logger.child({module: 'migrate'})
+        logger: log
     });
 
     try {
         await umzug.up();
         await sequelize.close();
     } catch (e) {
+        log.error(e);
```

Comment:
It was super ambiguous without this log when migration failed. We should notify user here.